services/submission.js

```diff
@@ -1618,83 +1618,108 @@ class Submission {
     }
 
     /**
-     * API: update the data-model version for the submission.
+     * API: update the submission info.
      * @param {*} params
      * @param {*} context
      * @returns {Promise<Submission>}
      */
-    async updateSubmissionModelVersion(params, context) {
+    async updateSubmissionInfo(params, context) {
         verifySession(context)
             .verifyInitialized();
 
-        const {_id, version, submitterID} = params;
+        const {_id, version, submitterID, submissionName} = params;
         const aSubmission = await this._findByID(_id);
         if(!aSubmission){
             throw new Error(ERROR.INVALID_SUBMISSION_NOT_FOUND);
         }
 
-        const [userScope, validVersions, { prevSubmitter, newSubmitter }] = await Promise.all([
-            this._getUserScope(context?.userInfo, USER_PERMISSION_CONSTANTS.DATA_SUBMISSION.REVIEW, aSubmission),
-            (async () => {
-                const dataModels = await this.fetchDataModelInfo();
-                return this._getAllModelVersions(dataModels, aSubmission?.dataCommons);
-            })(),
-            (async () => {
-                if (submitterID) {
-                    const newSubmitter = await this.userDAO.findFirst({id: submitterID});
-                    const preSubmitter = await this.userDAO.findFirst({id: aSubmission?.submitterID});
-                    return {prevSubmitter: preSubmitter, newSubmitter: newSubmitter};
-                }
-                return {};
-            })()
-        ]);
-
+        const userScope = await this._getUserScope(context?.userInfo, USER_PERMISSION_CONSTANTS.DATA_SUBMISSION.REVIEW, aSubmission);
```

Comment:
Next line (1637) will block submitters from using this API, because they won't have review permission, so userScope will be None.